curses.c

```diff
@@ -628,25 +627,11 @@ extern void mtr_curses_redraw(struct mtr_ctl *ctl)
 
   move(0, 0);
   attron(A_BOLD);
-  pwcenter("My traceroute  [v" PACKAGE_VERSION "]");
+  snprintf(buf, sizeof(buf), "%s%s%s", "My traceroute  [v", PACKAGE_VERSION, "]");
```

Comment:
Earlier string was read-only, and pwcenter() needs write access. Yep, minor thing but necessary when all warnings were removed.